charts/gitea/templates/deploymentconfig.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  generation: 1
+  labels:
+    app: {{ .Values.name | default "gitea" | quote }}
+    app.kubernetes.io/part-of: gitea
+    app.openshift.io/runtime: git
+  name: {{ .Values.name | default "gitea" | quote }}
+spec:
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    app: {{ .Values.name | default "gitea" | quote }}
+    deploymentconfig: {{ .Values.name | default "gitea" | quote }}
+  strategy:
+    activeDeadlineSeconds: 21600
+    resources: {}
+    type: Recreate
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: gitea
```

Comment:
this is `app: {{ .Values.name | default "gitea" | quote }}` in all other cases so probs best to align these, and better yet to use a helpers.tpl for this stuff

Reply:
@InfoSec812 ; only this left outstanding. as @springdo is on holiday, once this is done I'll merge and if there are any other changes needed, he can do PRs once hes back.